python/pyspark/sql/tests.py

```diff
@@ -2186,7 +2186,7 @@ def test_replace(self):
         # replace with subset specified with one column replaced, another column not in subset
         # stays unchanged.
         row = self.spark.createDataFrame(
-            [(u'Alice', 10, 10.0)], schema).replace(10, 20, subset=['name', 'age']).first()
+            [(u'Alice', 10, 10.0)], schema).replace(10, value=20, subset=['name', 'age']).first()
```

Comment:
Are the above two test changes necessary?

Reply:
I don't think it's necessary but let me keep them since at least it tests different combinations of valid cases.